# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/public/mew/fix-cite-config-bug.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---


This fixes a logic bug to make the cite behaviour configurable.

  - Rene Rebe <rene@exactcode.de>

--- mew-3.3/mew-draft.el	2003-06-19 05:58:13.000000000 +0000
+++ mew-3.3-fixed/mew-draft.el	2003-09-11 17:49:10.000000000 +0000
@@ -499,8 +499,8 @@
 	  (let ((mark-active t) (transient-mark-mode nil))
 	    (cond
 	     ;; arg will be effect in mew-cite-original
-	     ((and (mew-mark) (or (not mew-cite-ignore-mouse-region)
-				  (not (mew-mouse-region-p))))
+	     ((and (mew-mark) (and (not mew-cite-ignore-mouse-region)
+				   (not (mew-mouse-region-p))))
 	      (setq beg (region-beginning) end (region-end)))
 	     ((mew-header-p)
 	      ;; header exists in Message mode. Skip the header
